src/screens/WaitTimeScreen.tsx

```diff
@@ -1,42 +1,471 @@
+              onPress={() => {
+                setIsVisible(true);
+              }}
+            >
+              <StyledText style={styles.sortByText}>Sort By</StyledText>
+            </Button>
+            {poiTypes.map((type: string, index: number) => {
+              return (
+                <Tag
+                  key={index}
+                  style={styles.tags}
+                  styles={StylesOverride.tagStyles}
+                >
+                  {type}
+                </Tag>
+              );
+            })}
+          </ScrollView>
+        </View>
+        {modalComponent}
+        <View style={styles.stepsContainer}>
+          <Steps
+            styles={StylesOverride.stepsStyles}
+            size="small"
+            direction="horizontal"
+          >
+            {steps1.map((_item: any, index: any) => (
+              <Step key={index} status={"wait"} />
+            ))}
+          </Steps>
+        </View>
+        <View style={styles.stepTextContainer}>
+          {steps1.map((item: any, index: any) => (
+            <View style={styles.stepTextView} key={index}>
+              <StyledText style={styles.stepTitleText}>{item.title}</StyledText>
+              <StyledText style={styles.stepDescriptionText}>
+                {item.description}
+              </StyledText>
+            </View>
+          ))}
+        </View>
+        <View style={styles.stepScrollDivider} />
+        <ScrollView
+          refreshControl={
+            <RefreshControl refreshing={refreshing} onRefresh={onRefresh} />
+          }
+        >
+          <List>
+            {poiData.map((poi: any, index: any) => (
+              <Item
+                key={index}
+                onPress={() => {
+                  navigation.navigate(LOCATION_DETAILS, {
+                    location: poi.location,
+                  });
+                }}
+                extra={
+                  <View>
+                    <Brief style={styles.poiWaitTimeText}>
+                      {poi.waitTime + " mins"}
+                    </Brief>
+                  </View>
+                }
+                arrow="horizontal"
+                thumb={
+                  <View style={styles.poiIcon}>
+                    {renderIcon(poi.type, theme.name)}
+                  </View>
+                }
+              >
+                <StyledText style={styles.poiDistanceText}>
+                  {poi.distance + " km"}
+                </StyledText>
+                <StyledText>{poi.location}</StyledText>
+                <StyledText style={styles.poiLastUpdatedText}>
+                  {renderLastUpdated(poi.lastUpdated)}
+                </StyledText>
+              </Item>
+            ))}
+          </List>
+        </ScrollView>
+      </View>
+    </TouchableWithoutFeedback>
   );
 };
 
 const styles = StyleSheet.create({
-  huge: {
-    fontSize: 20,
+  searchContainer: {
+    paddingTop: 20,
+    backgroundColor: "#FFFFFF",
```

Comment:
All background colors & colors here _that change based on theme_ should be stored in the theme constants and pulled from there. Not sure which ones are changing or not.

Reply:
I think we need to address this first before using theme styles in stylesheets https://github.com/QueueTime/qtime-frontend/pull/47#discussion_r1083517363

Reply:
I'd like for sure to use theme styles in stylesheets (you can just use them directly if you add the styling to theme). The comment is more just a refactoring of how we import it.